<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Diuse</class>
 <widget class="QWidget" name="Diuse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ARK汉化补丁一键替换</string>
  </property>
  <widget class="QLineEdit" name="filePath">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>601</width>
     <height>20</height>
    </rect>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>671</width>
     <height>20</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="tipVersion">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>80</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="version">
   <property name="geometry">
    <rect>
     <x>604</x>
     <y>80</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>本地版本:</string>
   </property>
  </widget>
  <widget class="QLabel" name="onlineVersion">
   <property name="geometry">
    <rect>
     <x>604</x>
     <y>60</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>最新版本:</string>
   </property>
  </widget>
  <widget class="QLabel" name="tipOnlineVersion">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>60</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tip">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>571</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="path">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>文件路径:</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>100</y>
     <width>77</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="openPath">
      <property name="text">
       <string>选取路径</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Go">
      <property name="text">
       <string>一键替换</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="online">
      <property name="text">
       <string>更新补丁</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="renew">
      <property name="text">
       <string>更新数据</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="TieBa">
      <property name="text">
       <string>汉化大大</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
